pilot/pkg/features/pilot.go

```diff
@@ -120,6 +120,11 @@ var (
 	WorkloadEntryCrossCluster = env.Register("PILOT_ENABLE_CROSS_CLUSTER_WORKLOAD_ENTRY", true,
 		"If enabled, pilot will read WorkloadEntry from other clusters, selectable by Services in that cluster.").Get()
 
+	// For Gateway SDS, the 'config cluster' is already the default - no change.
+	CentralIstiodAccess = env.Register("PILOT_REMOTE_CLUSTER_AUTH", false,
+		"If disabled, pilot will ignore CLUSTER_ID node meta and clusterid header, only support clients and secrets from config cluster."+
+			"If enabled, workloads in other clusters can connect to Istiod and get sidecar secrets from any cluster.").Get()
```

Comment:
I have a great difficulty understanding the flag,  from the implement detail, it should be:

if disabled, luster id from node metadata and identity cannot match 

If enabled, cluster id from node metadata and identity must match 

So  `PILOT_REMOTE_CLUSTER_AUTH` is confusing

Reply:
Suggestion ? I renamed it to PILOT_REMOTE_CLUSTER_ACCESS - to indicate the remote clusters have access to this Istiod. 

Maybe PILOT_PRIMARY_REMOTE_AUTH ? The intent is to allow primary/remote mode when the user is using it, but not allow remotes when the user has one Istiod per cluster.

Reply:
No suggestions yet. Need to think about